src/core/components/accordion/accordionItem/accordionItem.tsx

```diff
@@ -0,0 +1,21 @@
+import { AccordionItem as RadixAccordionItem } from '@radix-ui/react-accordion';
+import classNames from 'classnames';
+import { forwardRef, type ReactNode } from 'react';
+
+export const AccordionItem = forwardRef<HTMLDivElement, { children: ReactNode; className?: string; value: string }>(
```

Comment:
Create an `interface IAccordionItemProps` for the properties as we do for every component.
Same for the `AccordionItemContent` and `AccordionItemHeader` components